rfhub2/cli/rfhub_importer.py

```diff
@@ -0,0 +1,216 @@
+from pathlib import Path
+import re
+from robot.errors import DataError
+from robot.libdocpkg import LibraryDocumentation
+import robot.libraries
+from typing import Dict, List, Set, Tuple
+
+from .api_client import Client
+from rfhub2.model import Collection, Keyword
+
+
+RESOURCE_PATTERNS = {'.robot', '.txt', '.tsv', '.resource'}
+ALL_PATTERNS = (RESOURCE_PATTERNS | {'.xml', '.py'})
+EXCLUDED_LIBRARIES = {'remote.py', 'reserved.py', 'dialogs.py', 'dialogs_jy.py',
+                      'dialogs_py.py', 'dialogs_ipy.py', 'setup.py'}
+INIT_FILES = {'__init__.txt', '__init__.robot', '__init__.html', '__init__.tsv'}
+
+
+class RfhubImporter(object):
+
+    def __init__(self, paths: Tuple[Path, ...], no_installed_keywords: bool, client: Client) -> None:
```

Comment:
I would reorder the params to make client first. You usually put this kind of flags like `no_installed_keywords` at the end of argument list

Reply:
Fixed